domains/certificates/Shared/DataContext/Migrations/20240701111541_RemovedSyncPosition.cs

```diff
@@ -0,0 +1,33 @@
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace DataContext.Migrations
+{
+    /// <inheritdoc />
+    public partial class RemovedSyncPosition : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropTable(
+                name: "SynchronizationPositions");
+        }
+
+        /// <inheritdoc />
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
```

Comment:
I dont believe this Down method is needed

Reply:
It's autogenerated. I don't neccesarily disagree with you. I tried reading on the subject and found this that suggest having it for development reasons: https://github.com/dotnet/efcore/issues/14382

Reply:
Talked with @martinhenningjensen as well and we agree on leaving it for integration tests purposes since it's autogenerated - not that we ever see it used. It's not a hill I wanna die on. I wouldn't spend much time on writing it myself thoough. What do you think?